Source/Core/Common/MemArenaUnix.cpp

```diff
@@ -0,0 +1,93 @@
+// Copyright 2008 Dolphin Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include <cerrno>
+#include <cstddef>
+#include <cstdlib>
+#include <cstring>
+#include <set>
+#include <string>
+
+#include <fcntl.h>
+#include <sys/mman.h>
+#include <unistd.h>
+
+#include "Common/CommonFuncs.h"
+#include "Common/CommonTypes.h"
+#include "Common/Logging/Log.h"
+#include "Common/MemArena.h"
```

Comment:
Since this file is implementing the class in `MemArena.h`, I think it would be best to have it at the top (that's what I did with #10270).  `MemArenaAndroid.cpp` already has it at the top, but `MemArenaWin.cpp` would need to be changed too.